test/bdd/fixtures/docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
       - ANCHOR_CREDENTIAL_ISSUER=http://orb2.domain1.com
       - ANCHOR_CREDENTIAL_URL=http://orb2.domain1.com/vc
       - ANCHOR_CREDENTIAL_SIGNATURE_SUITE=Ed25519Signature2018
-      - ANCHOR_CREDENTIAL_DOMAIN=domain1.com
+      - ANCHOR_CREDENTIAL_DOMAIN=https://witness1.example.com/ledgers/maple2021
```

Comment:
Is this a real (resolvable) URL?

Reply:
We are currently not using it, this is sample from spec. I thought it would be more realistic.